setup.py

```diff
@@ -8,6 +8,17 @@
   pass
 
 try:
+  from subprocess import STDOUT, check_call
+  import os
+  check_call(['apt-get', 'install', '-y', 'libhdf5-serial-dev'])
```

Comment:
Actually, my thought actually is to have an installation set of dependencies for travis and keep those in `.travis.yml`, and a separately deps.sh script in the repo that someone would run locally. Peregrine does something like this: https://github.com/swift-nav/peregrine/